### PR TITLE
add projects

### PR DESCRIPTION
**Pull request by sils**

### CODE REVIEW

File: data/projects.js [hunk 1]
```diff
@@ -81,5 +81,51 @@ coala should support generating metrics for your code.",
...
+		],
+		"mentors" : [
+			"Makman2",
+			"Udayan12167"
+		],
+		"initiatives" : ["GSoC"],
+		"tags" : ["Core", "Bears", "Debugging"],
+		"markdown": "debug_profile.md"
+	},
+	{
```
Comment (fneu) [inline data/projects.js]: why a second time though?

File: data/projects/debug_profile.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+In the development lifecycle of a Bear it needs to be debugged and profiled
+and coala should provide a comprehensive environment
+for this. In this project the participant shall develop a system to easily
+step into the analysis code (likely using the Pdb module already integrated
+into Python) and step out as soon as the analysis finished.
+
+On this way it would be useful retrieving profiling information while
```
Comment (fneu) [inline data/projects/debug_profile.md]: this sentence is kinda weird. Is it needed?

File: data/projects.js [hunk 3]
```diff
@@ -81,5 +81,51 @@ coala should support generating metrics for your code.",
 		"initiatives" : ["GSoC"],
 		"tags" : ["Core", "Bears"],
 		"markdown": "implement_metrics_for_coala.md"
+	},
+	{
+		"name" : "Debug and Profile Bears",
```
Comment (fneu) [inline data/projects.js]: I like this although I'm a bit sceptical about "profiling" as in "performance".
The overhead to running a linter that is currently introduced by coala is immense.

File: data/projects/nested_languages.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+Multiple programming languages can coexist in a single source file. coala would
+benefit from having an API to support such analysis algorithms. If we have e.g.
```
Comment (fneu) [inline data/projects/nested_languages.md]: API to support analysis algorithms as in "the analysis algorithms are using the API"?
wouldn't it make more sense to strip the file apart beforehand and pass the different parts to different analysis routines?

**Comment by sils** [comment]
@Udayan12167 continuing your PR here

**Comment by hemangsk** [comment]
ack c4b6d47 ed28417

**Comment by sils** [comment]
rebasing

**Comment by hemangsk** [comment]
ack 65d503d b277c9b

**Comment by sils** [comment]
reack https://github.com/coala/projects/pull/77/commits/b277c9b7c675dc31616c569585841ba2d1c6e931 https://github.com/coala/projects/pull/77/commits/65d503da8d5c7199475f5080db1f33144c0667da

**Pull request merged**